lib/screens/home/explorer/explorer_bloc.dart

```diff
@@ -0,0 +1,68 @@
+import 'dart:async';
+
+import 'package:get_it/get_it.dart';
+import 'package:provenance_wallet/services/models/provenance_validator.dart';
+import 'package:provenance_wallet/util/strings.dart';
+
+class ExplorerBloc extends Disposable {
+  @override
+  FutureOr onDispose() {
+    // TODO: implement onDispose
+    throw UnimplementedError();
+  }
+}
+
+class StakingDetails {
+  StakingDetails({
+    required this.delegates,
+    required this.validators,
+    this.selectedType = Strings.dropDownDelegate,
+    this.selectedStatus = Strings.dropDownAllStatuses,
+    required this.address,
+  });
+
+// FIXME: delegates are a different type eventually
+  List<ProvenanceValidator> delegates;
+  List<ProvenanceValidator> validators;
+  String selectedType;
+  String selectedStatus;
+  String address;
+}
+
+enum ValidatorType {
```

Comment:
Looking a [these docs](https://docs.provenance.io/ecosystem/community/delegator) and wondering if this should be renamed, since these aren't "types" of validators. Would this be more accurate?
```
enum DelegationState {
    bonded,
    redelegated,
    unbonded,
}
```

Reply:
Technically yes. I named them the way I did to match the endpoints so I could literally use `.toString()`. It's totally cool though since I'll just make another getter property on the extension.